### ISSUE TITLE
can not click seek to time on some video

### ISSUE THREAD

**Issue by vonhatduc** [Yotsubal/YDrive-Google-Drive-Proxy#2]
i want to fix it , please create a branch i wiil create pull request

 if (http_response_code() != '403') {
                if ($this->downloadThis) {
                        header('Pragma: public');
                        header('Cache-Control: must-revalidate, post-check=0, pre-check=0');
                        header('Content-Disposition: attachment; filename="'.$this->title.' [' . $this->v_res . 'p].mp4"');
                }

                $fp = fopen($source['link'], 'rb');

  
            $size   = $headers['Content-Length']; // File size
            $length = $size;           // Content length
            $start  = 0;               // Start byte
            $end    = $size - 1;       // End byte

            header('Content-Type: video/mp4');
            header('Accept-Ranges: bytes');
            header('Last-Modified: '.$headers['Last-Modified']);
            if (isset($_SERVER['HTTP_RANGE'])) {
                $c_start = $start;
                $c_end   = $end;
                if (strpos($_SERVER['HTTP_RANGE'], '=') == false) {
                    list(, $range) = explode(':', $_SERVER['HTTP_RANGE'], 2);
                } else {
                    list(, $range) = explode('=', $_SERVER['HTTP_RANGE'], 2);
                }
                if (strpos($range, ',') !== false) {
                    header('HTTP/1.1 416 Requested Range Not Satisfiable');
                    header("Content-Range: bytes $start-$end/$size");
                    exit;
                }
                if ($range == '-') {
                    $c_start = $size - substr($range, 1);
                }else {
                    $range  = explode('-', $range);
                    $c_start = $range[0];
                    $c_end   = (isset($range[1]) && is_numeric($range[1])) ? $range[1] : $size;
                }
                $c_end = ($c_end > $end) ? $end : $c_end;

                $start  = $c_start;
                $end    = $c_end;
                $length = $end - $start + 1;
                fseek($fp, $start);
                header('HTTP/1.1 206 Partial Content');
            }

            header("Content-Range: bytes $start-$end/$size");
            header("Content-Length: $length");

                while (!feof($fp)) {
                        echo fread($fp, 1024 * 1024 * 7);
                        flush();
                        ob_flush();
                }

                fclose($fp);

**Comment by thehoaj12345** [comment]
Muốn sửa j v

**Comment by vonhatduc** [comment]
click tua nhanh cos ddc dau cha

**Comment by vonhatduc** [comment]
them doan code cua tui vo la tua nhanh phim dc

**Comment by Yotsubal** [comment]
@vonhatduc I already create another branch, you can create pull request to the new branch

**Comment by SDInHearts** [comment]
Yes. Same problem. Please can you add this code into fluidplayer or plyr or videojs ?

**Comment by KayckMatias** [comment]
@vonhatduc I put your code and it didn't work on 720p videos, the video for when I advance, only on 480p and 360p videos, can you solve it?

**Comment by wulandrima1** [comment]
Yes. Same problem
